src/main/java/at/researchstudio/sat/merkmalservice/model/Standard.java

```diff
@@ -0,0 +1,37 @@
+package at.researchstudio.sat.merkmalservice.model;
+
+import java.util.Objects;
+
+public class Standard {
+    private final String id;
+    private final Organization organization;
+
+    public Standard(String id, Organization organization) {
+        Objects.requireNonNull(id);
+        Objects.requireNonNull(organization);
+        this.id = id;
+        this.organization = organization;
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public Organization getOrganization() {
+        return organization;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
only use id check in equals:
- otherwise sets will add another entry instead of replacing the existing one (e.g. for name changes)